CMakeLists.txt

```diff
@@ -613,6 +637,16 @@ if (LBANN_HAS_VTUNE)
   target_link_libraries(lbann PUBLIC ${VTUNE_STATIC_LIB})
 endif ()
 
+if(LBANN_HAS_CALIPER)
+  target_link_libraries(lbann PUBLIC adiak)
```

Comment:
```suggestion
  target_link_libraries(lbann PUBLIC adiak::adiak)
```

Reply:
namespace added in Adiak  version 0.2.2 which isn't in Spack preferred 0.2.1
install(EXPORT adiak-targets NAMESPACE adiak:: DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/adiak)

Yes, I ran into this before way back when for hand compiles. I'll look into making this consistent

Reply:
```suggestion
  if (TARGET adiak::adiak)
    target_link_libraries(lbann PUBLIC adiak::adiak)
  else ()
    target_link_libraries(lbann PUBLIC adiak)
  endif ()
```
Would this work?